test/services/projectList.spec.js

```diff
@@ -114,14 +114,20 @@ describe('services', () => {
         const service = new ProjectListService();
         expect(() => service.validateProjectListEntry({ path: '/Test/Path' })).toThrow(Error);
         // eslint-disable-next-line prettier/prettier
-        expect(() => service.validateProjectListEntry({ id: null, path: '/Test/Path' })).toThrow(Error);
+        expect(() => service.validateProjectListEntry({ id: null, path: '/Test/Path' })).toThrow(
```

Comment:
I'm okay to accept the linter suggestion here, but we should remove the eslint-disable line above